config/admins.txt

```diff
@@ -91,4 +91,8 @@ drovidicorv = Game Master
 Dunc = Game Master
 MMMiracles = Game Master
 bear1ake = Game Master
-CoreOverload = Game Master
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
this is the start of the "HEAD" version of this file.
you're still in the "fix conflicts" stage essentially.